tests/components/shelly/test_binary_sensor.py

```diff
@@ -53,26 +53,24 @@ async def test_block_binary_sensor(
     assert entry.unique_id == "123456789ABC-relay_0-overpower"
 
 
-async def test_block_binary_sensor_extra_state_attr(
+async def test_block_binary_sensor_creation(
```

Comment:
We have a test named `test_block_binary_sensor` https://github.com/home-assistant/core/blob/062965498e4d34aa7bdd7a6c4c9b6e9a1aff826f/tests/components/shelly/test_binary_sensor.py#L32 what is the difference in this test? I think this test can be removed, it doesn't increase any coverage